### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by santoshlite** [santoshlite/Beibo#2] [General]
Hey,

Any features you'd like to see in the Oracle? Any idea on how to improve it? Any question?

Just shoot! 🔫

**Comment by copypasteearth** [comment]
Could this possibly work for intraday trading?

**Reply by santoshlite** [reply]
Hey,

Sorry, it is not possible for the moment since Beibo uses yfinance's data that is not intraday.

**Reply by buckydroid** [reply]
Not sure if it's possible. But if you can add a forecasting graph output or if I give 30 days prediction as an input, you can print the prices of next 30 days instead of just returning the 30th day's return.

Great work by the way.

**Comment by stillakg** [comment]
@ssantoshp

 I have a few worthy suggestions : 

1) Please add implementation GARCH(x,y) model also for forecasting with mention of model type (x,y)
2) Please add auto ARIMA , ARIMA model type (x,y) when results are printed 
3) ARIMAX , SARIMAX implementation would be amazing
4) Please add implementation of Hidden Markov Model as well
5) Please add implementation of Poisson AR model

A few questions : 

1) Does the prophet model and other forecasting models account for weekends and bank holidays in respective country equity markets of which stocks are mentioned in the portfolio array ?
